src/pages/community/meetups.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Druid Community Meetups
+layout: simple_page
+canonical: 'https://druid.apache.org/community/meetups'
+---
+
+## Druid Community Meetups
+
+Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to regular
+meetups in cities all over the world.
+
+All Apache Druid Meetups are covered the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.
+
+### Speaker Guidelines
+
+**Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.
+
+**Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with – there may be a community member in the audience who can help you out! Engineering always involves trade-offs, so being realistic about them helps establish you as a credible advisor.
+
+**Focus on the Technical:** This audience understands and has built data architectures – if you get it, they’ll get it. Of course, start with the basics of who you are, what you built, and why, and then go all-in on the technical insights, examples, and demos.
```

Comment:
I like the Confluent "everyone with a mic should be technical" angle - should we strengthen that kind of view here?